### PR TITLE
Threads list view and individual thread views working, new database i…

### PR DESCRIPTION
**Pull request by kbooth1000**
…nitializer added 
   @janelleany @ponchieponcho

The threads list view (home page) and individual thread views are working (not styled).
• A couple of new dependencies were added, so after pulling to your local repo, type 'node install' in your terminal to update your node modules.
• Create a new database called townsquare_db and run the psql commands from the file 'create-initial-database.txt' to populate it with test data.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,48 +0,0 @@
-<!DOCTYPE html>
```
Comment (ponchieponcho) [inline public/index.html:1]: Is there a reason this file doesn't exist anymore?

File: views/createPost.dust [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{>"layout" /}
```
Comment (ponchieponcho) [inline views/createPost.dust:1]: What is .dust?

**Comment by kbooth1000** [comment]
dustjs is a templating engine that allows you to write basic html templates
that you can pass variables into (like the sql query results) to populate
fields in the views. Using that, I'm passing the results of the server's
route functions to views like thread.dust and threads.dust.

File: views/createPost.dust (hunk 2 above)
Comment (kbooth1000) [reply views/createPost.dust:1]: dustjs is a templating engine that allows you to write basic html templates
that you can pass variables into (like the sql query results) to populate
fields in the views. Using that, I'm passing the results of the server's
route functions to views like thread.dust and threads.dust.

File: public/index.html (hunk 1 above)
Comment (kbooth1000) [reply public/index.html:1]: I moved the code from that into the views/createPost file.

Comment (kbooth1000) [reply public/index.html:1]: forgot to mention that

Comment (kbooth1000) [reply public/index.html:1]: are you still working with that file? Didn't mean to mess you up.

Comment (ponchieponcho) [reply public/index.html:1]: It's fine but ill prob need to run through it with you tomorrow so I know what is going on

**Comment by kbooth1000** [comment]
Yeah. Sounds good.

**Review by ponchieponcho — APPROVED** [review]
Went over in person

**Pull request merged**